Art/aman/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style> 
+div {
+  width: 100px;
+  height: 100px;
+  background-color: red;
+  animation-name: example;
+  animation-duration: 4s;
+}
+
+@keyframes example {
+  from {background-color: red;}
+  to {background-color: yellow;}
+}
+</style>
```

Comment:
# Feedback - remove css from the HTML file
- [ ] please remove the css from the HTML file and use css file in which you will add your style - do not forget to link the stylesheet in your html afterwards